pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools", "numpy>=1.12", "scipy>=0.18.0", "Cython>=0.26"]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "celer"
+dependencies =[
```

Comment:
wdyt of adding doc requirements here and removing `doc_requirements.txt` in the `doc/` folder?

Reply:
Thought about it but it would have some side effects namely changing parts in documentation, readme, and CI/CD

What about tackling that and ``requirement.txt`` in a separate PR?

Reply:
as you see fit!